eventio/tools.py

```diff
@@ -28,22 +54,22 @@ def read_from(fmt, f):
     return result
 
 
-def read_ints(n, f):
+def read_ints(f, n_ints):
     ''' read n ints from file f '''
-    return read_from('{:d}i'.format(n), f)
+    return read_from(f, '{:d}i'.format(n_ints))
 
 
-def read_from_without_position_change(fmt, f):
+def read_from_without_position_change(f, fmt):
```

Comment:
This function seems to be implemented but never used. Was it a test of some kind and should be removed, or do you plan to use it later somewhere?

Reply:
This was used in the old version all over the place but is probably not needed anymore